app/controllers/interviews_controller.rb

```diff
@@ -13,6 +13,7 @@ def new
 
   def show
     @interview = Interview.find(params[:id])
+    @questions = @interview.template.questions
```

Comment:
Interview belongs_to: template and template has_and_belongs_to_many :questions.
Interview have relation with Question trough Template.
Here so if I understood you correctly